src/Marketplace/Interfaces/ConvertibleToSDKRequestsInterface.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Pagarme\Core\Marketplace\Interfaces;
+
+interface ConvertibleToSDKRequestsInterface
+{
+    public function convertMainToSDKRequest();
+    public function convertSecondaryToSDKRequest();
```

Comment:
Talvez a gente não precise dessa interface... Nem todas as coisas que são convertíveis em sdk request precisam ter um método que converte um "main" para sdk request e um "secondary" para sdk request, essa é uma necessidade exclusiva do split que tem recebedores principais e secundários. Existe alguma interface que force a implementação de um "convertToSdkRequest" em outros aggregates?

Reply:
Sim, o aggregate de Shipping tem uma interface que força ela implementar esse método. Mas, realmente parece que pra esse caso específico não é necessário uma interface. Removido!